File: 151003/Matoshko/tweet-service/src/main/java/services/tweetservice/domain/response/TweetResponseTo.java
package services.tweetservice.domain.response;

import java.time.LocalDateTime;
import java.util.List;

public record TweetResponseTo(
        Long id,
        Long creatorId,
        String title,
        String content,
        LocalDateTime created,
        LocalDateTime modified,
        List<MessageResponseTo> messageList,
        List<StickerResponseTo> stickerList) {
}

File: 151003/Mytnik/Lab2/Lab2/Lab2/DTO/TweetResponseTo.cs
using Lab2.DTO.Interface;
using Lab2.Models;

namespace Lab2.DTO
{
    public class TweetResponseTo(int Id, int? CreatorId, string? Title, string? Content, string? Created, string? Modified ,ICollection<Note> Notes, Creator? Creator) : IResponseTo
    {
        public int Id { get; set; } = Id;

        public int? CreatorId { get; set; } = CreatorId;

        public string? Title { get; set; } = Title;

        public string? Content { get; set; } = Content;

        public string? Created { get; set; } = Created;

        public string? Modified { get; set; } = Modified;

        public virtual ICollection<Note> Notes { get; set; } = Notes;

        public virtual Creator? Creator { get; set; } = Creator;
    }
}
